drivers/scheduler/k8s/k8s.go

```diff
@@ -3604,6 +3670,26 @@ func (k *K8s) ValidateVolumes(ctx *scheduler.Context, timeout, retryInterval tim
 			}
 
 			log.Infof("[%v] Validated group snapshot: %v", ctx.App.Key, obj.Name)
+		} else if obj, ok := specObj.(*pds.ModelsDeployment); ok {
+			ss, err := k8sApps.GetStatefulSet(obj.GetClusterResourceName(), *obj.Namespace.Name)
+			if err != nil {
+				return &scheduler.ErrFailedToValidateStorage{
+					App:   ctx.App,
+					Cause: fmt.Sprintf("Failed to get StatefulSet: %v , Namespace: %v. Err: %v", obj.Name, obj.Namespace, err),
+				}
+			}
+			// Providing the scaling factor in timeout
+			scalingFactor := *ss.Spec.Replicas
+			if *ss.Spec.Replicas > *ss.Spec.Replicas {
```

Comment:
can you check here seems u r  comparing same things

Reply:
done